pkg/install/crdmigration/preflight.go

```diff
@@ -382,3 +387,19 @@ func validateCRDsExist(ctx context.Context, logger logrus.FieldLogger, opt *Opti
 
 	return nil
 }
+
+func validateEtcdBackupConfiguration(ctx context.Context, logger logrus.FieldLogger, opt *Options) error {
+	logger.Info("Validating etcd backup configuration…")
+
+	for name, seed := range opt.Seeds {
+		if seed.Spec.BackupRestore != nil {
+			return fmt.Errorf("Seed %s uses the deprecated `backupRestore` configuration; please migrate to the `etcdBackupRestore` configuration before proceeding, as the deprecated options have been removed in KKP 2.20", name)
+		}
+
+		if seed.Spec.EtcdBackupRestore.DefaultDestination == nil {
```

Comment:
seed.Spec.EtcdBackupRestore == nil, no need for the check or migration as the backups are not even configured